packages/salesforcedx-vscode-apex-debugger/src/telemetry/telemetry.ts

```diff
@@ -65,4 +65,15 @@ export class TelemetryService {
     const hrend = process.hrtime(hrstart);
     return Number(util.format('%d%d', hrend[0], hrend[1] / 1000000));
   }
+
+  public sendEvent(msg: string, type: string): void {
```

Comment:
This function was added to specifically handle a metric event, why not embrace that. The other methods in this class are specializations as well that are handling a very specific need. Seems reasonable to apply the same here.

Reply:
Thanks for the suggestion.  I changed the function name from `sendEvent()` to `sendMetricEvent()` to be more specific about what it does.